sdk/core/Azure.Core/src/Shared/PemReader.cs

```diff
@@ -12,6 +12,12 @@ namespace Azure.Core
     /// <summary>
     /// Reads PEM streams to parse PEM fields or load certificates.
     /// </summary>
+    /// <remarks>
+    /// This class provides a downlevel PEM decoder since <c>PemEncoding</c> wasn't added until net5.0.
+    /// The <c>PemEncoding</c> class takes advantage of other implementation changes in net5.0 and,
+    /// based on conversations with the .NET team, runtime changes. This was based on <c>PemEncodiner</c>
+    /// and reviewed by @bartonjs on the .NET / cryptography team.
```

Comment:
Should we be putting GitHub aliases in ref docs?

Reply:
Perhaps not in rendered doc comments. As just a code comment sure. I think it's important to bring up here as bartonjs is the .NET security expert worth mentioning in this regard. Maybe I could refactor that a bit.